rust/repo/{{ cookiecutter.repo_name }}/deny.toml

```
@@ -19,6 +19,14 @@ allow = [
     "ISC",
     "MIT",
     "Unicode-DFS-2016",
+{%- if cookiecutter.license == "GPL-3.0-only" or cookiecutter.license == "AGPL-3.0-only" -%}
+    "GPL-3.0-only",
+    "GPL-3.0-or-later",
+{%- endif -%}
+{%- if cookiecutter.license == "AGPL-3.0-only" -%}
```

:wrench: suggest omitting trailing `-` to keep resultant array entries as multi-line
```suggestion
{%- if cookiecutter.license == "AGPL-3.0-only" %}
```